src/components/container/sticky-bar/sticky-bar.scss

```
@@ -0,0 +1,78 @@
+@use '../../core/styles' as sbb;
+
+:host {
+  @include sbb.box-sizing;
+
+  --sbb-sticky-bar-padding-block: var(--sbb-spacing-responsive-xxxs);
+  --sbb-sticky-bar-animation-duration: var(--sbb-animation-duration-5x);
+  --sbb-sticky-bar-animation-easing: var(--sbb-animation-easing);
+  --sbb-sticky-bar-intersector-height: #{sbb.px-to-rem-build(1)};
+  --sbb-sticky-bar-background-color: var(
+    --sbb-container-background-color,
+    var(--sbb-color-white-default)
+  );
+
+  display: contents;
+}
+
+:host([color='white']) {
+  --sbb-sticky-bar-background-color: var(--sbb-color-white-default);
+}
+
+:host([color='milk']) {
+  --sbb-sticky-bar-background-color: var(--sbb-color-milk-default);
+}
+
+:host([color='midnight']:not([data-settled])) {
```

data-settled is not existing? 
```suggestion
:host([color='midnight']) {
```